CMakeLists.txt

```diff
@@ -74,6 +74,8 @@ option(SILKWORM_CORE_ONLY "Only build Silkworm Core" OFF)
 option(SILKWORM_CLANG_COVERAGE "Clang instrumentation for code coverage reports" OFF)
 option(SILKWORM_SANITIZE "Build instrumentation for sanitizers" OFF)
 
+get_filename_component(SILKWORM_MAIN_DIR . ABSOLUTE)
```

Comment:
is this different from CMAKE_SOURCE_DIR?
https://cmake.org/cmake/help/latest/variable/CMAKE_SOURCE_DIR.html#variable:CMAKE_SOURCE_DIR

Reply:
It is if Silkworm is used as a sub-project of another CMake project.